.github/ISSUE_TEMPLATE/if-plugin-registry-form-template.md

```diff
@@ -0,0 +1,32 @@
+---
+name: IF Plugin Registry Form template
+about: Use this template to submit an IF plugin for registry
+title: "[PLUGIN REGISTRY]"
+labels: ''
+assignees: ''
+
+---
+
+## Plugin Name
+<!--- How do you call your plugin? -->
+
+## Plugin GitHub URL
+<!--- URL -->
+
+## Plugin Registry Basic Requirements
+- [ ] My plugin executes a demo manifest yml with no errors or crashes.
+- [ ] My plugin has README file.
+- [ ] My plugin code includes a sample manifest .yaml file.
+- [ ] My plugin's code contain unit tests that cover >=80% of the plugins code.
```

Comment:
I know I said that 1090% coverage is a bit too strict, but I also think 80% is a bit arbitrary. The important branches might all be in the uncovered 20% and besides,t he threshold is arbitrary. Maybe we just ask for *some* unit tests and later we can reward projects that achieve 100% coverage with a badge.